nibabel/nifti1.py

```diff
@@ -232,6 +233,17 @@
     (2003, 'rgb vector', (), "NIFTI_INTENT_RGB_VECTOR"),
     (2004, 'rgba vector', (), "NIFTI_INTENT_RGBA_VECTOR"),
     (2005, 'shape', (), "NIFTI_INTENT_SHAPE"),
+    # FSL-specific intent codes - FNIRT
```

Comment:
Sorry to be a pain - but do you have a reference for where these codes can be found?